echo-notifications/src/main/resources/templates/swabbie/body.ftl

```diff
@@ -138,6 +139,17 @@
               </td>
             </tr>
             </#if>
+            <tr>
+              <td class="loop__key" align="right" valign="top" style="padding: 4px 16px 4px 0; font-family: Helvetica, Arial, sans-serif; font-size: 12px; font-weight: bold;" width="30%">
+                Link
+              </td>
+              <td align="left" style="padding: 4px 0; font-family: Helvetica, Arial, sans-serif; font-size: 12px;">
```

Comment:
Per our chat, lets remove the specific link. In the notification context in swabbie, lets pass in a `resourceType` and a `resourceUrl` (iirc this field should be there already) i.e `resourceUrl: https://spinnaker/search?q=`

https://github.com/spinnaker/swabbie/blob/3440531305926e9cda41cd5b9e93445941e701dc/swabbie-core/src/main/kotlin/com/netflix/spinnaker/config/SwabbieProperties.kt#L92

Reply:
Agree . I changed the template to use the `spinnakerWebUrl` ( i think we still need this, its different from `resourceUrl` which links to the api route and not the web ui ) and use the `resourceType` we already have as part of the `notificationContext`

Reply:
`spinnakerweburl` is what `resourceUrl` was intended to be. It's not meant to be for api but for visualizing the resource in spinnnaker UI